src/pages/groupComparison/ClinicalData.tsx

```diff
@@ -327,7 +320,7 @@ export default class ClinicalData extends React.Component<IClinicalDataProps, {}
                         vertData={(this.vertAxisDataPromise.result! as IStringAxisData).data}
                         horzCategoryOrder={(this.horzAxisDataPromise.result! as IStringAxisData).categoryOrder}
                         vertCategoryOrder={(this.vertAxisDataPromise.result! as IStringAxisData).categoryOrder}
-                        categoryToColor={this.categoryToColor}
+                        categoryToColor={STUDY_VIEW_CONFIG.colors.reservedValue}
```

Comment:
I think the right way to handle this is to go into GroupComparisonStore and assign these colors to a group with the corresponding name. We would want those colors to be in every tab not just this one, right?

Reply:
yeah, true. I'll update